src/components/styled.tsx

```diff
@@ -42,12 +42,15 @@ const rotate = keyframes`
   }
 `;
 
+const animationSpinner = props => css `${rotate} 1s linear infinite;`;
+
 export const Loader: FC = (props) => (
   <ImSpinner2
     {...props}
     style={{
       verticalAlign: 'middle',
-      animation: `${rotate} 1s linear infinite`,
+        // @ts-ignore 
+      animation: animationSpinner,
```

Comment:
Yeah. I saw this problem. It looks like react icon use styled components below the hood. 

You can get around this (without errors) by abstracting the styled components instance:

```
const rotate = keyframes`
  from {
    transform: rotate(0deg);
  }

  to {
    transform: rotate(360deg);
  }
`;

const Loader = styled(ImSpinner2)`
  vertical-align: middle;
  animation: ${css`
    ${rotate} 1s linear infinite
  `};
`;
```